### PR TITLE
Add MySQL Enterprise doc. 

### PR DESCRIPTION
**Pull request by suaas21**

### CODE REVIEW

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+```console
+$ kubectl create ns demo
+namespace/demo created
+```
+
+> **Note:** YAML files used in this tutorial are stored in [docs/examples/day-2-operations](/docs/examples/day-2-operations) directory of [stashed/docs](https://github.com/stashed/docs) repository.
+
+### Apply Horizontal Scaling on MySQL Group Replication
+
+Here, we are going to deploy a  `MySQL` group replication using a supported version by `KubeDB` operator. Below section will check the supported `MySQL` versions.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:40]: Remove
>Below section will check the supported `MySQL` versions.

Add:
>At first, we are going to deploy a group replication server with 3 members. Then, we are going to add two additional members through horizontal scaling. Finally, we will  remove 1 members from the cluster again via horizontal scaling.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:41]: ### Prepare Group Replication Cluster
- **Find Supported MySQL Version**
- **Deploy MySQL Group Replication cluster**
- **Wait for the cluster to be ready**

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+8.0.14      8.0.14    kubedb/mysql:8.0.14      true         49s
+8.0.14-v1   8.0.14    kubedb/mysql:8.0.14-v1                49s
+8.0.18      8.0.18    kubedb/mysql:8.0.18                   49s
+8.0.19      8.0.19    kubedb/mysql:8.0.19                   49s
+8.0.20      8.0.20    kubedb/mysql:8.0.20                   49s
+8.0.3       8.0.3     kubedb/mysql:8.0.3       true         49s
+8.0.3-v1    8.0.3     kubedb/mysql:8.0.3-v1                 49s
+```
+
+The version above that does not show `DEPRECATED` `true` is supported by `KubeDB` for `MySQL`. Now we will select a version from `MySQLVersion` for `MySQL` group replication. For `MySQL` group replication deployment, we will select version `8.0.20`.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:74]: is -> are

Comment (hossainemruz) [reply docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:74]: Remove:
>Now we will select a version from `MySQLVersion` for `MySQL` group replication. For `MySQL` group replication deployment, we will select version `8.0.20`.

Add:
>You can use any non-deprecated version. Here, we are going to create a MySQL Group Replication cluster using MySQL  `8.0.20`.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+
+> **Note:** YAML files used in this tutorial are stored in [docs/examples/day-2-operations](/docs/examples/day-2-operations) directory of [stashed/docs](https://github.com/stashed/docs) repository.
+
+### Apply Horizontal Scaling on MySQL Group Replication
+
+Here, we are going to deploy a  `MySQL` group replication using a supported version by `KubeDB` operator. Below section will check the supported `MySQL` versions.
+
+**Find supported MySQL Version:**
+
+When you have installed `KubeDB`, it has created `MySQLVersion` crd for all supported `MySQL` versions. Let's check support versions,
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:44]: Let's check support versions -> Let's check the supported MySQL versions

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+8.0.20      8.0.20    kubedb/mysql:8.0.20                   49s
+8.0.3       8.0.3     kubedb/mysql:8.0.3       true         49s
+8.0.3-v1    8.0.3     kubedb/mysql:8.0.3-v1                 49s
+```
+
+The version above that does not show `DEPRECATED` `true` is supported by `KubeDB` for `MySQL`. Now we will select a version from `MySQLVersion` for `MySQL` group replication. For `MySQL` group replication deployment, we will select version `8.0.20`.
+
+#### Prepare Group Replication
+
+Now, we are going to deploy a `MySQL` group replication using version `8.0.20`.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:78]: Remove:
>Now, we are going to deploy a `MySQL` group replication using version `8.0.20`

Add:
>In this section, we are going to deploy a MySQL group replication cluster with 3 members. Then, in the next section we will scale-up the cluster using horizontal scaling.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+$ watch -n 3 kubectl get pod -n demo -l kubedb.com/kind=MySQL,kubedb.com/name=my-group
+Every 3.0s: kubectl get pod -n demo -l kubedb.com/kind=MySQ...  suaas-appscode: Tue Jun 30 22:45:33 2020
+
+NAME         READY   STATUS    RESTARTS   AGE
+my-group-0   1/1     Running   0          17m
+my-group-1   1/1     Running   0          14m
+my-group-2   1/1     Running   0          11m
+```
+
+Let's check the StatefulSet pods have formed a MySQL group replication,
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:143]: Let's verify that the StatefulSet's pods have joined into a group replication cluster,

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
++---------------------------+--------------------------------------+------------------------------+-------------+--------------+-------------+----------------+
+| CHANNEL_NAME              | MEMBER_ID                            | MEMBER_HOST                  | MEMBER_PORT | MEMBER_STATE | MEMBER_ROLE | MEMBER_VERSION |
++---------------------------+--------------------------------------+------------------------------+-------------+--------------+-------------+----------------+
+| group_replication_applier | 596be47b-baef-11ea-859a-02c946ef4fe7 | my-group-1.my-group-gvr.demo |        3306 | ONLINE       | SECONDARY   | 8.0.20         |
+| group_replication_applier | 815974c2-baef-11ea-bd7e-a695cbdbd6cc | my-group-2.my-group-gvr.demo |        3306 | ONLINE       | SECONDARY   | 8.0.20         |
+| group_replication_applier | ec61cef2-baee-11ea-adb0-9a02630bae5d | my-group-0.my-group-gvr.demo |        3306 | ONLINE       | PRIMARY     | 8.0.20         |
++---------------------------+--------------------------------------+------------------------------+-------------+--------------+-------------+----------------+
+```
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:161]: So, we can see that our group replication cluster has 3 members. Now, we are ready to apply horizontal scale on this group replication cluster.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+
+We are ready to apply horizontal scale on this group replication.
+
+#### Horizontal Scaling
+
+Here, we are going to scale up the number of server nodes of the `MySQL` group replication.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:167]: Here, we are going to add 2 members in our group replication cluster using horizontal scaling.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+
+**Create MySQLOpsRequest:**
+
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:170]: In order to scale your cluster, you have to create a MySQLOpsRequest CR with your desired number of members after scaling. Below is the YAML of the `MySQLOpsRequest` crd that we are going to create,

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 10]
```diff
@@ -0,0 +1,334 @@
...
+  type: HorizontalScaling  
+  databaseRef:
+    name: my-group
+  horizontalScaling:
+    member: 5
+```
+
+Here,
+
+- `spec.databaseRef.name` refers to the `my-group` MySQL object for operation.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:189]: `spec.databaseRef.name` specifies that we are performing operation on `my-group` MySQL databse.

File: docs/guides/mysql/scaling/horizontal-scaling/group_replication.md [hunk 11]
```diff
@@ -0,0 +1,334 @@
...
+- `spec.type` specifies that this is an `HorizontalScaling` type operation
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/group_replication.md:190]: - `spec.type` specifies that we are performing `HorizontalScaling` on our database.

File: docs/guides/mysql/scaling/horizontal-scaling/overview.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  docs_{{ .version }}:
+    identifier: my-horizontal-scaling-overview
+    name: Overview
+    parent: my-horizontal-scaling
+    weight: 20
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> :warning: **This doc is only for KubeDB Enterprise**: You need to be an enterprise user!
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/overview.md:13]: use `notice` shortcodes

File: docs/guides/mysql/scaling/horizontal-scaling/overview.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+<figure align="center">
+  <img alt="Stash Backup Flow" src="/docs/images/day-2-operation/ops_req-horizontal_scale.svg">
+<figcaption align="center">Fig: Horizontal scaling process of MySQL group replication</figcaption>
+</figure>
+
+The horizontal scaling process consists of the following steps:
+
+1. At first, a user creates a `MySQL` crd.
+
+2. `KubeDB` community operator watches for `MySQL` crd.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/overview.md:38]: for the MySQL CR

File: docs/guides/mysql/scaling/horizontal-scaling/overview.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+5. `KubeDB` enterprise operator watches for `MySQLOpsRequest`.
+
+6. When it finds a specific one, it pauses the `MySQL` object so that the `KubeDB` community operator doesn't perform any operation on the `MySQL` during the scaling process.  
+
+7. Then the `KubeDB` enterprise operator will scale the StatefulSet replicas to reach the expected number of nodes/members for the group replication.
+
+8. After successful scaling of the StatefulSet's replica, the `KubeDB` enterprise operator updates the `MySQL` object replicas.
+
+9. After successful scaling of the `MySQL` replicas, the `KubeDB` enterprise operator resumes the `MySQL` object so that the `KubeDB` community operator resumes it's actual operations.
```
Comment (hossainemruz) [inline docs/guides/mysql/scaling/horizontal-scaling/overview.md:52]: community operator can resume it's usual operations.

File: docs/guides/mysql/scaling/horizontal-scaling/overview.md [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+
+At each of the above steps, the `KubeDB` enterprise operator updates the `status` section of the `MySQLOpsRequest`.
```
[the remark this answers is not in the document]
Comment (hossainemruz) [reply docs/guides/mysql/scaling/horizontal-scaling/overview.md:54]: In the next doc, we are going to show a step by step guide on scaling of a MySQL group replication cluster using Horizontal Scaling.

File: docs/guides/mysql/upgrading/majorversion/standalone.md [hunk 16]
```diff
@@ -0,0 +1,341 @@
...
+    name: my-standalone
+  type: Upgrade
+  upgrade:
+    targetVersion: "8.0.21"
+```
+
+Here,
+
+- `spec.databaseRef.name` specifies that we are performing operation on `my-group` MySQL database.
+- `spec.type` specifies that we are going performing `Upgrade` on our database.
```
Comment (faem) [inline docs/guides/mysql/upgrading/majorversion/standalone.md:215]: *to perform

File: docs/guides/mysql/upgrading/majorversion/standalone.md [hunk 17]
```diff
@@ -0,0 +1,341 @@
...
+
+```console
+$ watch -n 3 kubectl get myops -n demo my-upgrade-major-standalone
+Every 3.0s: kubectl get myops -n demo my-up...  suaas-appscode: Wed Aug 12 16:12:03 2020
+
+NAME                          TYPE      STATUS       AGE
+my-upgrade-major-standalone   Upgrade   Successful   3m57s
+```
+
+We can see from the above output that the `MySQLOpsRequest` has succeeded. If you describe the `MySQLOpsRequest` you will see that the `MySQL`, `StatefulSet`, and its `Pod` have updated with a new image.
```
Comment (faem) [inline docs/guides/mysql/upgrading/majorversion/standalone.md:239]: you > we ?

File: docs/guides/mysql/upgrading/minorversion/standalone.md [hunk 18]
```diff
@@ -0,0 +1,341 @@
...
+
+Here, we are going to deploy a `MySQL` standalone using a supported version by `KubeDB` operator. Then we are going to apply upgrading on it.
+
+#### Prepare Group Replication
+
+At first, we are going to deploy a standalone using supported that `MySQL` version whether it is possible to upgrade from this version to another. In the next two sections, we are going to find out the supported version and version upgrade constraints.
+
+**Find supported MySQLVersion:**
+
+When you have installed `KubeDB`, it has created `MySQLVersion` cr for all supported `MySQL` versions. Let's check support versions,
```
Comment (faem) [inline docs/guides/mysql/upgrading/minorversion/standalone.md:49]: cr > CR?

File: docs/concepts/databases/mysql.md [hunk 19]
```diff
@@ -264,15 +279,57 @@ MySQL managed by KubeDB can be monitored with builtin-Prometheus and CoreOS-Prom
...
+
+  - `apiGroup` is the group name of the resource being referenced. The value for `Issuer` or   `ClusterIssuer` is "cert-manager.io"   (cert-manager v0.12.0 and later).
+  - `kind` is the type of resource being referenced. KubeDB supports both `Issuer`   and `ClusterIssuer` as values for this field.
+  - `name` is the name of the resource (`Issuer` or `ClusterIssuer`) being referenced.
+
+- `certificates` (optional) are a list of certificates used to configure the server and/or client certificate. It has the following fields:
+  
+  - `alias` represents the identifier of the certificate. It has the following possible value:
+    - `server` is used for server certificate identification.
+    - `archiver` is used for client certificate identification.
```
Comment (faem) [inline docs/concepts/databases/mysql.md:302]: client

**Pull request merged**